CHANGELOG.md

```diff
@@ -139,10 +77,46 @@ Updated the linter to `0.1.89`, which includes the following changes:
   This change only affects implementers of the `RegExp` interface; current
   code using Dart regular expressions will not be affected.
 
-#### `dart:isolate`
+### Language
 
-* **Breaking change:** The `await for` allowed `null` as a stream due to a bug
-  in `StreamIterator` class. This bug has now been fixed.
+*   **Breaking change:** Covariance of type variables used in super-interfaces
+    is now enforced (issue [35097][]).  For example, the following code was
+    previously accepted and will now be rejected:
+
+```dart
+class A<X> {};
+class B<X> extends A<void Function(X)> {};
+```
+
+* The identifier `async` can now be used in asynchronous and generator
+  functions.
+
+[35097]: https://github.com/dart-lang/sdk/issues/35097
+
+### Dart VM
+
+### Tools
+
+#### Linter
+
+The Linter was updated to `0.1.91`, which includes the following changes:
```

Comment:
If this is going to have linter 0.1.91, then it's missing a mention of the new lint `prefer_double_quotes` ([link](https://github.com/dart-lang/linter/commit/fd1ff984e54bc8c93767effe999f65bd970793a8))

Reply:
* New lint: `prefer_double_quotes` is in the list now, so this looks done.

Reply:
@athomas Looks like it's mentioned twice now.